apps/e2e-tests/x.spec.ts

```diff
@@ -0,0 +1,92 @@
+import { afterAll, beforeAll, describe, expect, test } from 'vitest';
```

Comment:
After we have this working at a basic level, (in a separate PR, of course) I'd like us to try 
:cucumber: https://cucumber.io/docs/installation/ for writing acceptance tests

Reply:
maybe